spec/components/radio_spec.rb

```diff
@@ -119,6 +119,34 @@ def component_name
     assert_select "legend h1", "What is your favourite skittle?"
   end
 
+  it "renders radio-group with a page heading caption" do
+    render_component(
+      name: "favourite-skittle",
+      heading: "What is your favourite skittle?",
+      heading_caption: "Question 3 of 9",
+      is_page_heading: true,
+      items: [
+        { label: "Red", value: "red" },
+        { label: "Blue", value: "blue" }
+      ]
+    )
+    assert_select ".govuk-radios"
+    assert_select "legend span.govuk-caption-xl", "Question 3 of 9"
```

Comment:
Same question here about checking the title renders ok with the caption provided